src/main/java/org/broadinstitute/consent/http/resources/DacResource.java

```diff
@@ -71,7 +71,12 @@ public Response createDac(@Auth AuthUser authUser, String json) throws Exception
         if (dac.getDescription() == null) {
             throw new BadRequestException("DAC Description is required");
         }
-        Integer dacId = dacService.createDac(dac.getName(), dac.getDescription());
+        Integer dacId;
+        if (Objects.isNull(dac.getEmail())) {
```

Comment:
unnecessary check? for clarity sake maybe it makes sense to just pass in the email here since it is null anyway. I see how on update you don't want to delete old emails if they are present, so the check makes sense there.

Reply:
I would like to keep the two separate code paths until we need to make email a required field.